experimental/check_gpu_ecc/check_gpu_ecc.py

```diff
@@ -95,6 +100,63 @@ def parse_nvidia_smi_gpu_id(ecc_d):
     return ecc_d
 
 
+def get_retired_pages_data():
+    GPU_RETIRED_PAGES_QUERY = "gpu_uuid,retired_pages.address,retired_pages.timestamp,retired_pages.cause"
+    cmd = ["nvidia-smi", "--query-retired-pages={}".format(GPU_RETIRED_PAGES_QUERY), "--format=csv,noheader"]
+    try:
+       cmdpipe = subprocess.Popen(cmd, stdout=subprocess.PIPE, stderr=subprocess.PIPE, encoding="utf-8")
+    except FileNotFoundError:
+       print("Error: Could not find the executable (nvidia-smi), make sure you have installed the Nvidia GPU driver.")
+       sys.exit(1)
+    rp_l = csv.reader(cmdpipe.stdout.readlines())
+
+    return list(rp_l)
+
+
+def get_datatime_obj(datetime_str):
+    return datetime.strptime(datetime_str, '%a %b %d %H:%M:%S %Y')
```

Comment:
Should this be `get_datetime_obj`?  It's a wrapper on a single line, why even create a function?

Reply:
corrected name.